spec/spec_helper.rb

```diff
@@ -99,11 +99,11 @@ def load_fixture(fixture)
     m = example.metadata[:file_path].match /spec\/v(?<version>\d+)\/.*/
     @api_version = m[:version]
     api = Object.const_get("DatadogAPIClient::V#{@api_version}")
-    @configuration = api.const_get("Configuration").new
-    @configuration.api_key["apiKeyAuth"] = ENV["DD_TEST_CLIENT_API_KEY"]
-    @configuration.api_key["appKeyAuth"] = ENV["DD_TEST_CLIENT_APP_KEY"]
-    @configuration.debugging = (!ENV["DEBUG"].nil? and ENV["DEBUG"] != false)
-    @api_client = api.const_get("ApiClient").new @configuration
+    @configuration = api::Configuration.new
+    @configuration.api_key = ENV["DD_TEST_CLIENT_API_KEY"]
+    @configuration.app_key = ENV["DD_TEST_CLIENT_APP_KEY"]
+    @configuration.debugging = (!ENV["DEBUG"].nil? and ENV["DEBUG"] != "false")
```

Comment:
This was a bug, since an environment variable can never contain `false`, since it's not a string :)